onnx/common/ir.h

```diff
@@ -847,6 +848,15 @@ friend struct Value;
   const std::vector<std::string>& initializer_names() {
     return initializer_names_;
   }
+  const int64_t get_initializer_index(Value* v) {
```

Comment:
use `std::find`, which will returns you an iterator (similar to a pointer) to the corresponding initializer (or end() which indicates not found).

Reply:
included this function because I need the tensor from a name; however initializers and initializers_name are separate lists.

Reply:
perhaps i should instead define a function getInitializerByName?

Reply:
Yeah you can just name it as getInitializer with `const std::string&` as input type